### PR TITLE
Bug 1213539 - (B2G 2.5) "Merge duplicates" header is not localized r=…

### PR DESCRIPTION
**Pull request by arcturus**
…ferjm

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a7cb7a9a38d21cca4533468cd15f77ecad9e6245

File: apps/communications/contacts/views/matching/js/matching_ui.js [hunk 1]
```diff
@@ -70,7 +70,8 @@
                                           'suggestedDuplicatesMessage',
                                           params);
         } else {
-          document.title = this._('duplicatesFoundTitle');
+          navigator.mozL10n.setAttribute(document.title,
```
Comment (julienw) [inline apps/communications/contacts/views/matching/js/matching_ui.js]: it should be `setAttributes`. This is breaking the merging window. Did one of you try the patch on the device ? :/

Comment (julienw) [inline apps/communications/contacts/views/matching/js/matching_ui.js]: is there no f*** test covering this code, not even unit test ???

Comment (julienw) [inline apps/communications/contacts/views/matching/js/matching_ui.js]: oh man, moreover `document.title` is not a DOM element...

**Pull request merged**